stable/minio/README.md

```diff
@@ -90,6 +90,10 @@ The following tables lists the configurable parameters of the Minio chart and th
 | `persistence.storageClass` | Type of persistent volume claim     | `generic`                                               |
 | `persistence.accessMode`   | ReadWriteOnce or ReadOnly           | `ReadWriteOnce`                                         |
 | `resources`                | CPU/Memory resource requests/limits | Memory: `256Mi`, CPU: `100m`                            |
+| `defaultBucket.enabled`    | If true, a bucket will be created after minio
+install | `false` |
+| `defaultBucket.name`       | Bucket name | ` ` |
+| `defaultBucket.policy` | Bucket policy | `Download` |
```

Comment:
The default should be `download` (lowercase).

Reply:
updated